### PR TITLE
Three tier build review

### PR DESCRIPTION
**Pull request by zednis**
related to https://jira.tw.rpi.edu/browse/DCO-182

### CODE REVIEW

File: src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java [hunk 1]
```diff
@@ -117,40 +109,28 @@ public AdditionsAndRetractions  process(
...
-            //System.out.println("Edit exsiting ones");
-            changes = editExistingStatements(configuration, submission);
-            //System.out.println(changes.toString());
-        } else {
-            //System.out.println("Create new ones");
-            changes = createNewStatements(configuration, submission );
-            //System.out.println(changes.toString());
-
-		    /* DCO-ID generation, by Han Wang (wangh17@rpi.edu) */
-            generateDcoIdN3(changes, submission);
```
Comment (mrpatrickwest) [inline src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java:145]: Is this code being called from someplace else? Generating our DCOId is necessary.

**Comment by mrpatrickwest** [comment]
Of course we'll push this to udco and verify that it's all working properly
1. Make sure DCOId is being created
2. Make sure that the dataset/distribution creation and file upload is still working
3. Make sure that the DOI entry for publications is still working

**Comment by zednis** [comment]
Agreed.  I am confident that the unit tests do not reflect whether any of those 3 features still work.

File: src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java (hunk 1 above)
Comment (mrpatrickwest) [inline src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java:145]: This method that's been commented out is a duplicate method from the one starting on line 87 just without one of the parameters. And this happens throughout the code and is why some methods are commented out.

**Comment by zednis** [comment]
@mrpatrickwest is this pull request ready to merge?

**Pull request merged**